applications/Covid Tracker/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width"> <!--to create a responsive site--> 
+        <meta name="description" content="Tracking covid cases around me"> 
+        <meta name="keywords" content="covid19 cases, tracking covid cases, covid tracker"> <!--for SEO--> 
+        <meta name="author" content="Rendell">
```

Comment:
Not a big deal, but you should remove this, since everyone in the class will be adding to this file. Don't worry, your name will be on the commit showing that you created this 😉

Reply:
Oh yes,  I've already learned html and CSS and I created this for practice. I just thought of sharing this with others for them to practice as well... I'll make the changes Mark.
It's just a dummy so I don't care about the name 😅